src/components/DeleteModalUI.tsx

```diff
@@ -0,0 +1,48 @@
+const MODAL_MESSAGES = {
+  confirmDeleteAll: 'Are you sure you want to delete all tasks?',
+  noTasksToDelete: 'No tasks to delete.',
+  closeButton: 'Close',
+  deleteButton: 'Delete',
+};
+
+interface DeleteModalUIProps {
+  handleDeleteAll: () => Promise<void>;
+  handleCloseModal: () => void;
+  hasTasks: boolean;
+}
+
+function DeleteModalUI({
+  handleDeleteAll,
+  handleCloseModal,
+  hasTasks,
+}: DeleteModalUIProps) {
+  return (
+    <div className="h-full w-full flex flex-col justify-center items-center gap-10">
+      <p>
+        {hasTasks
+          ? MODAL_MESSAGES.confirmDeleteAll
+          : MODAL_MESSAGES.noTasksToDelete}
```

Comment:
🪡 Nitpick: move this into a variable. Call it something like `message`

Reply:
updated with variable `message`